pom.xml

```diff
@@ -30,9 +30,11 @@
 
     <modules>
         <!--        <module>build-tools</module>-->
+        <module>config-server</module>
+        <module>discovery-server</module>
+
```

Comment:
redundant new line?

Reply:
No, it's a separation between  CORE services and SPRING CLOUD services